packages/web/src/components/SequenceView/SequenceMarkerMissing.tsx

```diff
@@ -24,9 +24,10 @@ export function SequenceMarkerMissingUnmemoed({ seqName, missing, pixelsPerBase,
   const { begin, end } = missing // prettier-ignore
 
   const id = getSafeId('missing-marker', { seqName, ...missing })
-  const x = begin * pixelsPerBase - pixelsPerBase / 2
   let width = (end - begin) * pixelsPerBase
   width = Math.max(width, BASE_MIN_WIDTH_PX)
+  const halfNuc = Math.max(pixelsPerBase, BASE_MIN_WIDTH_PX) / 2 // Anchor on the center of the first nuc
+  const x = begin * pixelsPerBase - halfNuc
```

Comment:
Let's take the missing range (N nucs) as an example.

I take a max of "pixels per nuc" and "min nuc width", which gives the width of a single nuc for this particular marker in pixels.

Then I take half of this number, which gives a half of a nuc.

I subtract this half-nuc-width from the x position of the rectangle representing the range.

Elements in SVG are anchored on their left edges by default. So subtracting half a nuc shifts the rectangle by half a nuc to the left. This way it looks as if it is anchored at the center of the first nuc in the range.